runcoms/.eslintrc.json

```diff
@@ -0,0 +1,149 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020
+},
+"env": {
+    "es6": true
+},
```

Comment:
is this how your prettier setup sets this file? example:
```
{
  "foo": {
    "bar": true
},
...
}

Reply:
lmfao, no this is 💯 my mistake. Nice catch.